### PR TITLE
Add DB setup instructions to README

### LINKED ISSUE #43: Extend DB setup description in README

### PR DESCRIPTION
**Pull request by student-tomasz**
Closes #43

### CODE REVIEW

**Comment by kamildebicki** [comment]
:+1:

**Comment by Kacper3331** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ environment and a PostgreSQL installed. On macOS you can install it using
 And on Debian based distros:
 
     sudo apt-get update
-    sudo apt-get install postgresql postgresql-contrib
+    sudo apt-get install postgresql postgresql-contrib libpq-dev
```
Comment (gotar) [inline README.md:24]: +1

**Comment by gotar** [comment]
you can merge

**Pull request merged**